cmd/hamctl/command/root.go

```diff
@@ -34,7 +45,7 @@ func NewRoot(version *string) (*cobra.Command, error) {
 		PersistentPreRunE: func(c *cobra.Command, args []string) error {
 			// all commands but version and completion requires the "service" flag
 			// if this is one of them, skip the check
```

Comment:
Classic comment issue. It tells what the code does, and now the code changed 🙈🤥

I guess we should remove it or update it.